core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BootMonitors.scala

```diff
@@ -38,23 +45,31 @@ object BootMonitors extends LazyLogging {
                    useWorkflowCollectionField: Boolean,
                    useWorkflowCollectionLabel: Boolean,
                    defaultBackend: CromwellBackend,
-                   methodConfigResolver: MethodConfigResolver)(implicit cs: ContextShift[IO]): Unit = {
+                   methodConfigResolver: MethodConfigResolver,
+                   avroUpsertPubSubProject: GoogleProject,
+                   avroUpsertPubSubTopic: String,
+                   avroUpsertPubSubSubscription: String,
+                   avroUpsertBucketName: String,
+                   avroUpsertBatchSize: Int)(implicit cs: ContextShift[IO]): Unit = {
     //Reset "Launching" workflows to "Queued"
-    resetLaunchingWorkflows(slickDataSource)
+//    resetLaunchingWorkflows(slickDataSource)
```

Comment:
ignore these commented out lines for now. i wanted a back rawls that didn't do all of these other things so i just commented it out